portality/bll/services/article.py

```diff
@@ -144,13 +147,29 @@ def create_article(self, article, account, duplicate_check=True, merge_duplicate
         # or an update
         is_update = 0
         if duplicate_check:
-            duplicate = self.get_duplicate(article, account.id)
+            duplicate = self.get_duplicate(article)
             if duplicate is not None:
-                if merge_duplicate:
-                    is_update  = 1
-                    article.merge(duplicate) # merge will take the old id, so this will overwrite
+                if update_article_id is None:
+                    if merge_duplicate:
+                        is_update = 1
```

Comment:
if (acount.has_role("admin")):
# merge whatever
else:
if duplicate.owner is not account
  raise exception
else:
merge